README.md

```diff
@@ -61,7 +60,7 @@ author.save(function(err){
 What is module.exports and why do we use it?
 
 ```text
-
+Module.exports takes all functions related to a specific line of code and places it into a file.
```

Comment:
`module.exports` allows us to export certain pieces of code (that we as developers choose) and reference it in other JS files.